department-of-veterans-affairs/abd-vro
# Update environment variables for redis containers

## What was the problem?

The environment variables for platform images have changed now that we have switched to using Bitnmami based images. This caused issues with our existing images which expected different behavior. Also, some of my prior changes to support the move to RDS had been removed so this restores those. 

Associated tickets or Slack threads:
- #1007 

## How does this fix it?[^1]

Moves to using the environment variables configured for use by the Bitnami docker images for `redis`. `rabbitmq` changes will be in followup PR to help with testing some issues that came up in integration tests.

## How to test this PR
- Tested with deployments to `dev` 

[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
